spec/system/admin_split_details_create_spec.rb

```diff
@@ -17,6 +17,20 @@
     login
   end
 
+  context 'if GITHUB_ORGANIZATION ENV is set' do
+    before do
+      ENV['GITHUB_ORGANIZATION'] = 'test'
```

Comment:
we should use our `with_env` helper here so that we unset this after the test case runs. it looks like `with_env('THING' => 'VALUE') do ... end`